src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metrics/PerformanceAnalyzerMetrics.java

```diff
@@ -60,6 +61,8 @@
   public static final String MASTER_CURRENT = "current";
   public static final String MASTER_META_DATA = "metadata";
   public static final String METRIC_CURRENT_TIME = "current_time";
+  public static final String SHARD_PRIMARY = "p";
+  public static final String SHARD_REPLICA = "r";
```

Comment:
I am wondering if a better place for these constants will be an `enum` within AllMetrics.java

Reply:
Changed it to AllMetrics